dbms/src/DataStreams/TiRemoteBlockInputStream.h

```diff
@@ -193,6 +200,8 @@ class TiRemoteBlockInputStream : public IProfilingBlockInputStream
         execution_summaries.resize(source_num);
         connection_profile_infos.resize(source_num);
         sample_block = Block(getColumnWithTypeAndName(toNamesAndTypes(remote_reader->getOutputSchema())));
+        if constexpr (is_streaming_reader)
+            decoder_ptr = std::make_unique<CHBlockChunkDecodeAndSquash>(sample_block, 8192);
```

Comment:
Maybe we can change to `context.getSettingsRef().max_block_bytes`?
I don't know the reason for using the magic number 8192..

Reply:
It comes from previous SquashingBlockInputStream's parameter. Not sure its original source。

Reply:
Extract a new constexpr,  not moved to Settings, for not seeing the situation that users/us will tune this parameter for performance usage.

Reply:
ok